### ISSUE TITLE
Clarify "optionally" https://smarthealth.cards#covid19

### ISSUE THREAD

**Issue by p9g** [dvci/vaccine-credential-ig#160]
The IG should be clear about when to use credential type https://smarthealth.cards#covid19

A verifier who only supports covid immunization or testing may want to look for that credential type and only continue verification if that is present.  Should the IG say that if vaccineCode is related to covid that credential type https://smarthealth.cards#covid19 SHALL be present in the SHC?  Similarly, if the Observation is for a covid-related test, credential type SHALL include https://smarthealth.cards#covid19.  Credential type https://smarthealth.cards#covid19 SHALL NOT be present if the immunization or observation is not related to covid.

We should probably have value sets to express the codes that mean "covid-related" for both Immunization and Observation.

**Comment by masnick** [comment]
I agree this should be clarified, but I'm not sure the FHIR IG should be describing this as it's outside the `fhirBundle`.

@jmandel what do you think about adding to the content on https://spec.smarthealth.cards/vocabulary/ to provide stronger guidance to implementers on how to populate `vc.type` and associate the types with FHIR IG(s)?
